docs/preview/features/powershell/azure-api-management.md

```diff
@@ -44,3 +44,19 @@ Creates a new API operation on the API Management service with using a specific
 PS> Create-AzApiManagementApiOperation -ResourceGroup $ResourceGroup -ServiceName $ServiceName -ApiId $ApiId -OperationId $OperationId -Method $Method -UrlTemplate $UrlTemplate -OperationName $OperationName -Description $Description -PolicyFilePath $PolicyFilePath
 # New API operation '$OperationName' on API Management service was added.
 ```	
+
+## Import policy to an API in the API Management service
+
+Imports a policy from a file to an API in Azure API Management.
```

Comment:
I'd indicate this is a base-policy/policy that applies to all operations within that API, just to make the difference between this command and the operation-policy command more explicit.
```suggestion
Imports a base-policy from a file to an API in Azure API Management.
```